arbnode/node.go

```diff
@@ -576,7 +581,13 @@ func createNodeImpl(
 		}
 	} else if l2Config.ArbitrumChainParams.DataAvailabilityCommittee {
 		return nil, errors.New("a data availability service is required for this chain, but it was not configured")
-	} else if config.EigenDA.Enable {
+	}
+
+	if config.EigenDA.Enable && config.DataAvailability.Enable && !config.BatchPoster.EnableEigenDAFailover {
+		return nil, errors.New("eigenDA and anytrust cannot both be enabled without EnableEigenDAFailover=true in batch poster config")
```

Comment:
Does config.DataAvailability.Enable mean enable anytrust?
Also why do you need both enabled to do failover? Can't you only have eigenda enabled to start with and then switch over the anytrust being enabled afterwards? Then you could enforce the invariant that only one DA layer is enabled at any one point.

Reply:
This is part of the ArbOS system config which is only globally updatable by a l2 system tx invoked via the rollup manager. Having both enabled is key for failover in a way which is understood by the validator software